releases/release-1.31/release-notes/maps/pr-125163-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 125163
+releasenote:
+  text: 'DRA: The Dynamic Resource Allocation (DRA) driver''s daemonset must be deployed
```

Comment:
```suggestion
  text: 'DRA: The Dynamic Resource Allocation (DRA) driver's daemonset had to be deployed
```

Reply:
I think in the context of this note, it is ok to keep the tense of the sentence present. It is a requirement going forward for the users of DRA drivers ds. Thoughts @cloudmelon @npolshakova ?